src/enums/zone_constants.ts

```diff
@@ -0,0 +1,35 @@
+import { COLOR_MAP } from 'enums/interventions';
+
+export interface LevelInfo {
+  level: string;
+  upperLimit: number;
+  name: string;
+  color: string;
+  detail: string;
+}
+
+export enum Level {
+  LOW = 'LOW',
+  MEDIUM = 'MEDIUM',
+  HIGH = 'HIGH',
+  UNKNOWN = 'UNKNOWN',
+}
+
+export const COLOR_ZONE = {
+  [Level.LOW]: COLOR_MAP.GREEN.BASE,
+  [Level.MEDIUM]: COLOR_MAP.ORANGE.BASE,
+  [Level.HIGH]: COLOR_MAP.RED.BASE,
+  [Level.UNKNOWN]: COLOR_MAP.GRAY.BASE,
+};
```

Comment:
Maybe we should get rid of `COLOR_ZONE` and use `COLOR_MAP` everywhere instead? In the per-metric files above we are no using `COLOR_ZONE`, we might as well just replace them.

I think that the colors we associate to high, medium, and low risk go beyond the scope of the 'zone' charts, so it makes sense that we pick the colors from a more centralized file.

Reply:
i'm also punting on this.